src/EPR.SubsidiaryBulkUpload.Application/Services/BulkUploadOrchestration.cs

```diff
@@ -23,6 +23,12 @@ public BulkUploadOrchestration(IRecordExtraction recordExtraction, ISubsidiarySe
         _notificationService = notificationService;
     }
 
+    public async Task NotifyStart(UserRequestModel userRequestModel)
```

Comment:
This method is missing a test - something like this will do:

```
    [TestMethod]
    public async Task Should_Notify_Start()
    {
        // Arrange
        var userId = Guid.NewGuid();
        var organisationId = Guid.NewGuid();
        var userRequestModel = new UserRequestModel { UserId = userId, OrganisationId = organisationId };

        var orchestrator = new BulkUploadOrchestration(_recordExtraction.Object, _subsidiaryService.Object, _bulkSubsidiaryProcessor.Object, _notificationService.Object);

        // Act
        orchestrator.NotifyStart(userRequestModel);

        // Assert
        _notificationService.Verify(ns => ns.SetStatus(It.IsAny<string>(), It.IsAny<string>()), Times.Once());
    }

```

Reply:
updated.

Reply:
I remember I added comments but those comments are not shown any more